examples/kafka/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
These are already present in for the root Gradle Wrapper and might have been added by accident / forgotten to be deleted from the inital project genarator.